### DISCUSSION TITLE
Kong Konnect Gateway requires both Basic Auth and API Key

### DISCUSSION THREAD

**Discussion by rahulrai-in** [Kong/kong#7780] [Help]
I installed the Konnect runtime and created a user for the backend service. I published the service to the developer portal and selected key-auth for authentication.

Strangely, just using the API Key to make API requests always returned HTTP 401 Unauthorized response from the gateway. I created a basic auth credential and used the coded credential in my request to have the request reach the backend service as so:

```
curl --location --request GET 'http://127.0.0.1/' \
--header 'apikey: APIKEY' \
--header 'Authorization: Basic TOKEN'
```

I want to understand whether this is the desired behavior and what is the rationale for having two credentials in the same request?

**Comment by dndx** [comment]
Hello @rahulrai-in,

First thing I would look at is to double check the `APIKEY` is actually correct. If the API key was provided incorrectly then it would behave as if there is no API key presented at all.

Also, you should make sure basic auth is not enabled for the Route/Service object that you are using for proxying this request (if it is not needed).

**Reply by rahulrai-in** [reply]
Thanks for the reply.

I checked my API key for correctness. The only plugins on my route are set due to enabling app registration. Here is the screenshot of the plugins.

![Screenshot_20210901-210033](https://user-images.githubusercontent.com/11886524/131661159-c5e1a15d-b0c6-4b9d-997e-21a49e11a05c.png)


Are there any other ways of enabling authentication that I might have missed?
